_events/2016-05-summit-boulder.md

```diff
@@ -11,6 +11,8 @@ poster_hero: "/img/media/boulder.jpg"
 
 location_section: true
 papers_section: true
+
+featured: true
```

Comment:
Have you checked what the front page looks like with the `featured` flag enabled? There's no thumbnail yet, so I suspect it looks broken.